ibmdbpy/statistics.py

```diff
@@ -762,17 +762,21 @@ def corr(idadf):
     #print(result_df)
 
     table_name = idadf.internal_state.current_state
-    args_string = "('intable=" + table_name + " , incolumn= "+column_string+" , outtable=" + table_name + "_temp');"
-    create_corr_query = "CALL nza..CORRELATION1000MATRIX" + args_string
-    corr_df = idadf.ida_query(create_corr_query)
+    #args_string = "('intable=" + table_name + " , incolumn= "+column_string+" , outtable=" + table_name + "_temp');"
```

Comment:
A general comment:
The cov-function still requires the SQL Extension Toolkit. So it would be good to redefine it as well.